litespi/gen.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+
+#
+# This file is part of LiteSPI.
+#
+# Copyright (c) 2015-2022 Florent Kermarrec <florent@enjoy-digital.fr>
+# Copyright (c) 2022 Victor Suarez Rovere <suarezvictor@gmail.com>
+# Copyright (c) 2020 Xiretza <xiretza@xiretza.xyz>
```

Comment:
Xiretza and Stefan contributed features for LiteEth that are not present here, please just keep you and me here.

Reply:
done, just take into account I based this source on the LiteEth one